### PR TITLE
test: Add additional unit tests to time series domain

### PR DESCRIPTION
**Pull request by bemwen**
## Description

The purpose of this PR is to add some missing unit tests to the time series domain

### CODE REVIEW

File: source/GreenEnergyHub.TimeSeries/source/GreenEnergyHub.TimeSeries.Tests/Application/Handlers/TimeSeriesCommandHandlerTests.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    {
+        [Theory]
+        [InlineAutoMoqData]
+        public async Task HandleAsync_WhenCalled_DispatchesCommandAndReturnsSuccessfulResult(
+            [Frozen] [NotNull] Mock<IMessageDispatcher<TimeSeriesCommand>> dispatcher,
+            [NotNull] TimeSeriesCommand command,
+            [NotNull] TimeSeriesCommandHandler sut)
+        {
+            // Arrange
+            var dispatched = false;
```
Comment (BjarkeMeier) [inline source/GreenEnergyHub.TimeSeries/source/GreenEnergyHub.TimeSeries.Tests/Application/Handlers/TimeSeriesCommandHandlerTests.cs:42]: Why not use https://github.com/Moq/moq4/wiki/Quickstart#verification?

**Review by Mech0z — APPROVED** [review]

File: source/GreenEnergyHub.TimeSeries/source/GreenEnergyHub.TimeSeries.Tests/Infrastructure/Messaging/Serialization/JsonMessageDeserializerTests.cs [hunk 2]
```diff
@@ -85,6 +90,35 @@ public async Task FromBytesAsync_CreatesMessage([NotNull] IInboundMessage expect
...
+            var deserialized = false;
+            serializer.Setup(
+                    s => s.DeserializeAsync(
+                        It.IsAny<Stream>(),
+                        It.IsAny<Type>()))
+                .Returns(
+                    new ValueTask<object>(
+                        await Task.FromResult(message.Object)
+                            .ConfigureAwait(false)))
+                .Callback<Stream, Type>(
```
Comment (BjarkeMeier) [inline source/GreenEnergyHub.TimeSeries/source/GreenEnergyHub.TimeSeries.Tests/Infrastructure/Messaging/Serialization/JsonMessageDeserializerTests.cs:111]: Why not use https://github.com/Moq/moq4/wiki/Quickstart#verification?

**Review by BjarkeMeier — APPROVED** [review]
LGTM, I'm however interested why we're not using MOQ verification for simpler and more intuitive tests?

File: source/GreenEnergyHub.TimeSeries/source/GreenEnergyHub.TimeSeries.Tests/Application/Handlers/TimeSeriesCommandHandlerTests.cs (hunk 1 above)
Comment (bemwen) [reply source/GreenEnergyHub.TimeSeries/source/GreenEnergyHub.TimeSeries.Tests/Application/Handlers/TimeSeriesCommandHandlerTests.cs]: Changed to use verify

File: source/GreenEnergyHub.TimeSeries/source/GreenEnergyHub.TimeSeries.Tests/Infrastructure/Messaging/Serialization/JsonMessageDeserializerTests.cs (hunk 2 above)
Comment (bemwen) [reply source/GreenEnergyHub.TimeSeries/source/GreenEnergyHub.TimeSeries.Tests/Infrastructure/Messaging/Serialization/JsonMessageDeserializerTests.cs]: Changed to use verify

**Pull request merged**